challenge-13/submissions/atplay90/solution-template.go

```
@@ -0,0 +1,201 @@
+        return nil, err
+    }
+    
+    // Test the connection
+    if err = db.Ping(); err != nil {
+        return nil, err
+    }
+    
+	
+	// Create the products table if it doesn't exist
+	// The table should have columns: id, name, price, quantity, category
+	_, err = db.Exec("CREATE TABLE IF NOT EXISTS products (id INTEGER PRIMARY KEY, name TEXT, price REAL, quantity INTEGER, category TEXT)")
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)